README.md

```diff
@@ -4,6 +4,67 @@ This project uses Quarkus, the Supersonic Subatomic Java Framework.
 
 If you want to learn more about Quarkus, please visit its website: https://quarkus.io/ .
 
+## Set up the app
+
+### Fork the Repository
+For this repository to your GitHub account.
+
+### Configure JIRA Lottery
+1. Write a valid `jira-issue-lottery.yml` config in the `.github` directory in your forked repository.
+    - **Valid Config Example:**
+      ```yaml
+      delay: P14D
+      participants:
+        - username: The-Huginn
+          projects:
+            - project: WFLY
+              components: [Logging]
+              maxIssues: 5
+            - project: ELY
+              components: [HTTP, Core]
+              maxIssues: 3
+      ```
+   `delay`::
+   The duration of the lottery cycle before participants are selected again.
+   + Duration as a String in ISO-8601 format, mandatory, no default.
+   
+   `participants`::
+   A list of participants with their preferences for participating in the lottery.
+   + List of Participants, mandatory, no default.
+   
+   `username`::
```

Comment:
I believe this is not yet reflected in main (my fault sorry), but this will most likely end up being an email address instead. We should just keep this in mind and I will try to keep it updated and suggest the change in #40 .

Reply:
Should I wait until #40 is merged and then edit this PR, or should I change it to an email address now?

Reply:
@The-Huginn can fix this in #40